src/note-db.ts

```diff
@@ -105,10 +105,20 @@ const addNoteToDb = (note: Note) => {
     vscode.window.showInformationMessage('Annotation saved!');
 };
 
+const getTODOFromSelectedText = (): string | undefined => {
+    const editor = vscode.window.activeTextEditor;
+    const selectedText = editor?.selection ? editor.document.getText(editor.selection) : '';
+    const todoSelector = /\/\/ TODO: (.*)\n/;
```

Comment:
Please remove the `\n`, it's a too strict rule. For it to work not only the line would need to be selected, but the line break too, which I think is less usual.